scripts/sct_compute_hausdorff_distance.py

```diff
@@ -267,7 +267,8 @@ def __init__(self, im1, im2=None, param=None):
     def compute_dist_2im_2d(self):
         nx1, ny1, nz1, nt1, px1, py1, pz1, pt1 = get_dimension(self.im1)
         nx2, ny2, nz2, nt2, px2, py2, pz2, pt2 = get_dimension(self.im2)
-        assert px1 == px2 and py1 == py2 and px1 == py1
+
+        assert round(px1, 2) == round(px2, 2) and round(py1, 2) == round(py2, 2) and round(px1, 2) == round(py1, 2)
```

Comment:
You can break in multiple lines this if you want.

Reply:
Wow. Strongly recomment to use `np.isclose()` here!

Reply:
Done! Thank you!